Dockerfile

```diff
@@ -1,81 +1,47 @@
 # Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
 #
-# This file is part of the IoT Agent for the LigthweightM2M protocol (LM2M) component
+# This file is part of tlightweightM2M-iotagent
 #
-# IOTA-LM2M is free software: you can redistribute it and/or
+# lightweightM2M-iotagent is free software: you can redistribute it and/or
 # modify it under the terms of the GNU Affero General Public License as
 # published by the Free Software Foundation, either version 3 of the License,
 # or (at your option) any later version.
 #
-# IOTA-LM2M is distributed in the hope that it will be useful,
+# lightweightM2M-iotagent is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 # See the GNU Affero General Public License for more details.
 #
 # You should have received a copy of the GNU Affero General Public
-# License along with IOTA-LM2M.
+# License along with lightweightM2M-iotagent.
 # If not, see http://www.gnu.org/licenses/.
 #
 # For those usages not covered by the GNU Affero General Public License
-# please contact with TeF IoT Team
-
-FROM centos:7
-
-MAINTAINER TeF IoT Team
+# please contact with: [daniel.moranjimenez@telefonica.com]
+#
+# Modified by: Daniel Calvo - ATOS Research & Innovation
+#
+ARG  NODE_VERSION=8.12.0-slim
```

Comment:
If dockefile is included in this PR I understand the following lines should be added to CHANGES_NEXT_RELEASE file:

```
Add: use NodeJS 8 in Dockerfile
Fix: use PM2 in Dockerfile
```

Reply:
Fixed with 41edfcdf6b4da736051e4894ce69f5622376da9a